xhr/resources/inspect-headers.py

```diff
@@ -1,9 +1,18 @@
+import os
+from six import PY3
+
 def get_response(raw_headers, filter_value, filter_name):
     result = ""
-    for line in raw_headers.headers:
-        if line[-2:] != '\r\n':
-            return "Syntax error: missing CRLF: " + line
-        line = line[:-2]
+    if PY3:
+        header_list = (os.linesep.join(
+            [s for s in raw_headers.as_string().splitlines() if s])).split('\n')
+    else:
+        header_list = raw_headers.headers
+    for line in header_list:
+        if not PY3:
```

Comment:
This seems wrong.

Reply:
Is it okay to be more specific? raw_headers derives from different classes in Python 2 and Python 3. Here is to handle them differently.

Reply:
In particular, it seems like the code isn't executed at all in Python 3.

Reply:
raw_headers derives from different classes in python 2 and python 3. In Python 3, HTTPMessage doesn't have 'headers" attribute or equivalent as we described in the commit message (see the reference as well). That's why this part code is not executed in Python 3.

Reply:
We can't just drop testing coverage. We probably need a better solution to how we deal with header logic migration in that case.

Reply:
Here a few new lines added for python 3 to perform similar functionality. It basically hasn't touched/removed any original python 2 code.

Reply:
Where does the Python 3 code test for a missing CRLF?